Applications/SimpleShot/task_runner.cpp

```diff
@@ -119,16 +131,75 @@ using LayerHandle = std::shared_ptr<ml::train::Layer>;
  * @param variant "one of UN, L2N, CL2N"
  * @return std::unique_ptr<ml::train::Model>
  */
+int find_model_name(std::string backbone)
+{
+	
+	   const char * a= backbone.c_str();
```

Comment:
I think the indentation seems wrong. Could you check once again?

Reply:
Could you use a proper variable name instead of just a?

Reply:
Please Check now